### ISSUE TITLE
Syncnet Training

### ISSUE THREAD

**Issue by ghost** [MRzzm/DINet#28]
I reproduced for training syncnet, what is the loss of syncnet that could take into train clip?
currently, it's around 0.21-0.25

**Comment by zhangziliang04** [comment]
Hi,@primepake I will do the samethings, can you share you syscnet code? this is my email: 1106110976@qq.com

**Comment by zhangsanfeng86** [comment]
Hi @primepake , Could you share the train code of syscnet? my email is :peterxjm.de@gmail.com

**Comment by roopeshn28** [comment]
Hi @primepake  Could you share the train code of syscnet ?
my email is :roopeshn1994@gmail.com

**Comment by saudnr33** [comment]
Hi @primepake Could you share the train code of syscnet ?
my email is :[saudnr33@gmail.com](mailto:saudnr33@gmail.com)

**Comment by huangxin168** [comment]
Hi @primepake Could you share the train code of syscnet ?
my email is :[huangxin888@gmail.com](mailto:huangxin888@gmail.com)

**Comment by primepake** [comment]
hi all, I added my code to training syncnet with original repo here: https://github.com/primepake/DINet

**Comment by lcc157** [comment]
Hi @primepake  Could you share the train code of syscnet ?
my email is [1040424772@qq.com](url)

**Comment by jiayueshiliuqi** [comment]
Hi @primepake  Could you share the train code of syscnet ?
my email is 1223998091@qq.com

**Comment by Bebaam** [comment]
Hey @primepake ,
could you please give some insights about your training?
Using BCE Loss as in Wav2Lip and using data with sync-corrected videos (confidence >6) I still can't reach better than a loss of 0.69, while with the same data the original Wav2Lip syncnet training works just fine. 
1.) Did you use sync corrected videos? And English only?
2.) Did you modify the DINet provided syncnet model itself?
3.) How much data did you use and after how many epochs did the value get below 0.69?

Thanks in advance.

**Comment by hyungraelee** [comment]
Hi @primepake Could you share the train code of syncnet?
my email is [hyunlee.dev@gmail.com](url)

**Comment by ZYBOBO** [comment]
Hello @primepake,  I am reproducing DINet and need to retrain the syncnet for the clip stage training. Could you please share the train code of syncnet?
My email is 570764061@qq.com.
Hope for your reply~

**Comment by ldincredible** [comment]
Hi @primepake Could you share the train code of syncnet?
my email is lqdincredible@gmail.com.
Hope for your reply~

**Comment by decajcd** [comment]
Hi @primepake Could you share the train code of syncnet?
my email is 1048062792@qq.com.
Hope for your reply~

**Comment by ghost** [comment]
hello everyone, I will make it public here: https://github.com/primepake/better_wav2lip

**Comment by tailangjun** [comment]
> hello everyone, I will make it public here: https://github.com/primepake/better_wav2lip

I want to use Wav2lip's syncnet to replace the syncnet in DINet, but there is no 64x64 128x128 256x256 resolution in Wav2lip. Will this be supported in the future?

**Comment by shahidmuneer** [comment]
could you please email your syncnet training code to shahidmuneer@g.skku.edu ?

**Comment by fenghe12** [comment]
Hi @primepake Could you share the train code of syncnet?
my email is [2678713685@qq.com](mailto:2678713685@qq.com).
Hope for your reply~
